ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/snmp.md

```diff
@@ -81,15 +81,16 @@ interface Management1
 | VRF | Status |
 | --- | ------ |
 | default |  Disabled  |
-| mgt |  Enabled  |
+| MGMT |  Enabled  |
 
 ### SNMP Hosts Configuration
 
 | Host | VRF | Username | Authentication level | SNMP Version |
 | ---- |---- | -------- | -------------------- | ------------ |
-| 10.6.75.99 | mgt | USER-READ | auth | 3 |
-| 10.6.75.99 | mgt | USER-WRITE | auth | 3 |
-| 10.6.75.121 | mgt | USER-READ | auth | 3 |
+| 10.6.75.121 | MGMT | - | - | 1 |
```

Comment:
I know this is still in draft, so just a few suggestions :)
You should add community to this table and render the community for 1/2c